sections/monte_carlo/quantum_monte_carlo/quantum_monte_carlo.tex

```diff
@@ -69,24 +61,246 @@ \subsubsection{Amplitude Estimation}\label{sec:MC_AE}
+In case the last qubits are $\ket{1}, \ket{0}$ for $\ket{\phi_1}, \ket{\phi_0}$ respectively, then the reflection would be
+\begin{equation}\label{eq:PhiReflection}
+  S_{\psi_0} = 1-2\ket{\psi_0}\ket{0}\bra{\psi_0}\bra{0},
+\end{equation}
+and can in turn be realized by applying a Z-gate to the last qubit.\\
+The second reflection can be implemented as $AR_{0}A^{\dagger} = AR_0A$ with $R_0$
+\begin{equation}\label{eq:ZeroReflection}
+  R_0 = 2\ket{0^n}\bra{0^n} - \mathbb{1}.
+\end{equation}
+ $AR_0A$ puts a minus sign in front of states orthorgonal to $\ket{U}$. The product of both reflection is then (thanks to Grover) a rotation of angle
+ $2\theta$ in the two-dimensional picture introduced above. \\
+ The following amplitude amplification procedure increases the amplitude of the good state close to 1:
+\begin{enumerate}
+  \item{Setup the starting state $\ket{U} = A\ket{0^n}$}.
+  \item{Repeat the following $\mathcal{O}(1 \sqrt{a})$}.
+  \begin{itemize}
+    \item Reflect through bad state $\ket{\phi_0}$ ($R_G$).
+    \item Reflect through $\ket{U}$ ($AR_0A$).
+  \end{itemize}
+\end{enumerate}
+After k iterations the state is 
+\begin{equation}\label{eq:Qktimes}
+\mathcal{Q}^k \ket{U} = \sin \biggl[ (2k+1)\theta_a \biggr] \ket{\phi_1} + \cos \biggl[ (2k+1)\theta_a \biggr] \ket{\phi_0}.
+\end{equation}
+We would like to end up with an angle $(2k+1)\theta_a \approx \pi/2$ to be close to $\sin(pi/2) \approx 1$.\\
+The above explanation describes how amplitude amplification is applied to the initial state $\ket{\phi}$ in order to amplify the good state $\ket{phi_1}$.
+But how is this actually connected to Quantum Phase Estimation which then gives a estimation $\tilde{a}$?\\
+Given again the statevector which is defined to be living in a two-dimensional subspace $H_{\phi}$: $\ket{\phi} \in H_{\phi}$ spannend by two orthogonal and normalized
+projectors $\ket{\phi_1}, \ket{\phi_0}$ onto $H_1$ and $H_0$.\\
+The action of $\mathcal{Q} := AR_0AR_g$ is given by
+\begin{align}\label{eq:actionOfQ}
+  \mathcal{Q} \ket{\phi_0} &= (2 \cos^2(\theta_a) - 1 )\ket{\phi_0} + 2 \sin(\theta_a)cos(\theta) \ket{\phi_1}\\
+  \mathcal{Q} \ket{\phi_1} &= -2 \sin(\theta_a) \cos(\theta_a) \ket{\phi_0} 
 \end{align}
-As we can see in \ref{eq:classic_M} and \ref{eq:AE_M} quantum computer improved the number error with the same number of samples
-\subsubsection{Construct the payoff function}\label{sec:MC_Payoff}
-A lot of payoff function can be constructed by piece-wise linear functions, in this section we will go over how to construct the piece-wise linear function in to quantum computer.
+Thus $mathcal{Q}$ corresponds to rotation by $2\theta_a$ in $H_\phi$
+\begin{equation}\label{eq:rotationMatrix}
+  \mathcal{Q} = 
+\begin{pmatrix}
+      cos(2\theta_a) & -sin(2 \theta_a) \\
+      sin(2\theta_a) & cos(2 \theta_a) 
+\end{pmatrix},
+\end{equation}
+which has the well has eigenvalues $\lambda_{\pm} = e^{\pm i \theta_a}$.\\
```

Comment:
Shouldn't the eigenvalues be $\lambda_{\pm} = e^{\pm i 2 \theta_a}$
So with 2\theta_a. Since the rotation is with 2theta and not with theta.

I have seen both in Literature. The following original paper uses 2:
Brassard, G., Høyer, P., Mosca, M., &#38; Tapp, A. (2000). Quantum Amplitude Amplification and Estimation. (https://arxiv.org/abs/quant-ph/0005055)

The following paper about option pricing uses 1:
Rebentrost, P., Gupt, B., &#38; Bromley, T. R. (2018). Quantum computational finance: Monte Carlo pricing of financial derivatives. (https://journals.aps.org/pra/abstract/10.1103/PhysRevA.98.022321)